web/api/v1/api.go

```diff
@@ -475,6 +486,74 @@ func (api *API) queryRange(r *http.Request) (result apiFuncResult) {
 	}, nil, res.Warnings, qry.Close}
 }
 
+func (api *API) queryExemplars(r *http.Request) apiFuncResult {
+	start, err := parseTime(r.FormValue("start"))
+	if err != nil {
+		err = errors.Wrapf(err, "invalid parameter 'start'")
```

Comment:
One year later, we now have invalidParamError and parseTimeParam in the api package